app/models/robots/bed/base.rb

```diff
@@ -47,11 +46,7 @@ def barcode
     def load(barcodes)
       # Ensure we always deal with an array, and any accidental duplicate scans are squashed out
       @barcodes = Array(barcodes).map(&:strip).uniq.reject(&:blank?)
-      @plates = if @barcodes.present?
-                  Sequencescape::Api::V2::Plate.find_all({ barcode: @barcodes }, includes: plate_includes)
-                else
-                  []
-                end
```

Comment:
Don't think you need to reverse this one - think this was an optimisation made by @JamesGlover to avoid an unnecessary API call to SS in the case where there are no barcodes to search on?